api-samples/action/demo/index.js

```diff
@@ -123,6 +122,55 @@ document
     showBadgeText();
   });
 
+// --------------------------
+// get/set badge text color
+// --------------------------
+async function showBadgeTextColor() {
+  const color = await chrome.action.getBadgeTextColor({});
+  document.getElementById('current-badge-txt-color').value = JSON.stringify(
+    color,
+    null,
+    0
+  );
+}
+
+showBadgeTextColor();
+
+document
+  .getElementById('set-badge-txt-color-button')
+  .addEventListener('click', async () => {
+    // To show off this method, we must first make sure the badge has text
+    let currentText = await chrome.action.getBadgeText({});
+    if (!currentText) {
+      chrome.action.setBadgeText({ text: 'hello :)' });
+      showBadgeText();
+    }
+
+    // Next, generate a random RGBA color
+    const color = [0, 0, 0].map(() => Math.floor(Math.random() * 255));
+
+    // Use the default background color ~10% of the time.
+    //
+    // NOTE: Alpha color cannot be set due to crbug.com/1184905. At the time of writing (Chrome 89),
+    // an alpha value of 0 sets the default color while a value of 1-255 will make the RGB color
+    // fully opaque.
+    if (Math.random() < 0.1) {
+      color.push(0);
+    } else {
+      color.push(255);
+    }
+
+    chrome.action.setBadgeTextColor({ color });
+    showBadgeTextColor();
+  });
+
+  document
+  .getElementById('reset-badge-txt-color-button')
+  .addEventListener('click', async () => {
+    chrome.action.setBadgeTextColor({ color: "#000000" });
```

Comment:
Can we use single quotes here? As a note, we now have prettier. Installing an extension like https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode in your text editor should allow you to consistently format things.